### DISCUSSION TITLE
Slither headache

### DISCUSSION THREAD

**Discussion by astrotimi** [Cyfrin/security-and-auditing-full-course-s23#218] [Q&A]
i'm a windows user and i have tried all possible means to install & run slither on my Vscode to no avail , somebody help!!!

**Comment by EngrPips** [comment]
what was the issue you encountered while trying to install & run slither.

**Reply by astrotimi** [reply]
I have used different methods but this is the error from slither extension on vscode

Error: 'slither' is not recognized as an internal or external command,
operable program or batch file

**Reply by EngrPips** [reply]
Hello @astrotimi, Follow the installation guide on the Slither official repo https://github.com/crytic/slither to install Slither properly.

**Reply by astrotimi** [reply]
Thanks but did that , done all that ,I ran all the instructions on my vscode , slither ain't working. I probably need a video that shows the exact full process cause GitHub ain't clear enough.
